MonoGame.Framework/Graphics/SpriteBatcher.cs

```diff
@@ -224,7 +224,16 @@ public void DrawBatch(SpriteSortMode sortMode, Effect effect)
 
             // Determine how many iterations through the drawing code we need to make
             int batchIndex = 0;
-            int batchCount = _batchItemList.Count;
+            int batchCount = _batchItemList.Count;
+
+            if (_device._enableMetrics)
+            {
+                unchecked
+                {
+                    _device._metrics._batchCount += (ulong)batchCount;
```

Comment:
So you would then use `SpriteCount` and `DrawCount` to understand how many sprites are being drawn and in how many draws that results in.  If you draw count starts to equal your sprite count you know that maybe you've got a small batch problem.

Or should we and an explicit `SpriteDrawCount` for counting draws from `SpriteBatch` vs draw calls from non-`SpriteBatch` stuff?